### DISCUSSION TITLE
Feature Request: Allow for brushes to be imported from Procreate/Affinity

### DISCUSSION THREAD

**Discussion by GottaHaveCoffee** [keiwando/toonsquid#156] [Ideas]
Hello,  I have a ton of brushes that I have purchased for procreate and a few from Affinity Designer. I would like to be able to import them and use them within Toonsquid. Currently, I do not think it is possible as I have tried to import both my brushes from Procreate and Affinity. I know you are able to make your own but some brush sets that I have purchased are ones that I cannot easily create myself but would love to use. Is this possible and if so, can you consider add the ability to import procreate brushes?

**Comment by dreeze12** [comment]
up

**Comment by fuki98** [comment]
YES!

**Comment by keiwando** [comment]
Unfortunately, the issue is that Procreate brushes that use shape or grain sources from the default source library (which is all default brushes and a lot of custom brushes) do not include images of those textures in the brush files that you export, which means that they can only be imported back into Procreate.

I haven't looked into the Affinity brush format yet but I'm also getting a lot more requests for Procreate brushes and better Photoshop brush support (e.g. regarding the dual brush functionality), so Affinity brushes have a very low priority at the moment.

**Comment by fuki98** [comment]
Aw damn. :( Cool to know we still have PS if that's what you're implying.
